_msprimemodule.c

```diff
@@ -2193,6 +2193,42 @@ RecombinationMap_init(RecombinationMap *self, PyObject *args, PyObject *kwds)
     return ret;
 }
 
+static PyObject *
+RecombinationMap_position_to_mass(RecombinationMap *self, PyObject *py_position)
```

Comment:
The standard convention here is the make the second parameter ``*args``, since this is actually the args Python tuple, not a single Python object.